repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -218,7 +218,11 @@ private void processOutput(List<BuildToBeInspected> listOfBuilds) {
                 BufferedWriter writer = new BufferedWriter(new FileWriter(outputPath));
 
                 for (BuildToBeInspected buildToBeInspected : listOfBuilds) {
-                    writer.write(buildToBeInspected.getBuggyBuild().getId() + "");
+                    if (this.config.getLauncherMode() == LauncherMode.REPAIR) {
+                        writer.write(buildToBeInspected.getBuggyBuild().getId() + "");
+                    } else {
+                        writer.write(buildToBeInspected.getBuggyBuild().getId() + "," + buildToBeInspected.getPatchedBuild().getId());
```

Comment:
You should use here the constant defined in Repairnator Core for the COMMA

Reply:
Done